Dockerfile

```diff
@@ -11,18 +11,27 @@ RUN pip install -r requirements.txt
 COPY ./jobs /jobs
 
 # this didn't solve the permission issues for celery.
-# for now running everything as root. 
+# for now running everything as root.
 # RUN chown nobody: /jobs && chmod u+rwX /jobs
 
-# copy radiance libraries
-COPY ./radiance/bin /usr/local/bin
-COPY ./radiance/lib /usr/local/lib/ray
-
 # copy ladybug and honeybee libraries.
 COPY ./ladybug ./ladybug
 COPY ./honeybee ./honeybee
 
 # copy all the python files for the app
 COPY ./*.py ./
 
-EXPOSE 5000
\ No newline at end of file
+# copy radiance libraries
+#COPY ./radiance/bin /usr/local/bin
+#COPY ./radiance/lib /usr/local/lib/ray
+
+# Install Radiance from source
+# Avoids issue you had here with missing lib file(?)
+RUN cd /tmp \
```

Comment:
Not copying lib files over was my mistake in copy-pasting the files over. I will get that fixed! Thank you for finding the issue and reporting it.

Downloading from GitHub is not really the best option for libraries that we already have locally. It increases the time for no good reason.

Speaking of best practices we should keep Radiance libraries which won't change frequently on the top of the docker file. This way docker will use the cache for the libraries even if we make changes to ladybug and honeybee which will be much more frequent.

I'm referencing this guy for both of these suggestions: https://youtu.be/ZVaRK10HBjo?t=1h26m31s

Reply:
Ah yes absolutely agree about caching radiance stuff in the earlier layers of the docker image. Poor practice on my behalf. And yep time + download also makes sense with regard to dockerfile build if importing external libraries. Glad it's all fixed :)

Reply:
Ah yes absolutely agree about caching radiance stuff in the earlier layers of the docker image. Poor practice on my behalf. And yep time + download also makes sense with regard to dockerfile build if importing external libraries. Glad it's all fixed :)